src/components/Breadcrumb/Breadcrumb.jsx

```diff
@@ -115,7 +115,6 @@ const Breadcrumb = ({ children, className, hasOverflow, ...other }) => {
                     {...child.props}
                     title={child.props.children}
                     key={`${child.props.children}-${i}`}
-                    primaryFocus={i === 0}
```

Comment:
What was this for?

Reply:
This is the old way to set primary focus element. Is now deprecated and Taylor added the `selectorPrimaryFocus` to the OverflowMenu

Reply:
Nope. The first item is focused by default now when the menu opens